connections/src/app/qr-save-mock/qr-code-save/qr-code-save.component.ts

```diff
@@ -11,15 +11,15 @@ import { Router } from '@angular/router';
 })
 export class QrCodeSaveComponent {
 
-  instagramHandle: string = '';
+  urlHandle: string = '';
   qrData: string | null = null;
   caption: string = '';
 
   constructor(private router: Router, private http: HttpClient) {}
 
   generateQrCode() {
-    const instagramUrl = `https://www.instagram.com/${this.instagramHandle}/`;
-    this.qrData = instagramUrl;
+    const urlHandle = `https://${this.urlHandle}/`;
```

Comment:
@Siddharth-Tiwari1712 How many differnt date values, or strings, or triggers can we put into this?

Like when a user scans a code... the code can do...
Send a line to us. so we can then trigger an update to a quest, and at the same time, do a separate thing, like send the user to another guests profile page, or to a follow page, or to a Page saying, yes, your scanned the "front of the room, explore more places" or

Reply:
We can discuss on this.